test/functional/p2p_encryption.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+# Copyright (c) 2018 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Tests NODE_ENCRYPTED (v2 protocol, encryption after BIP151)."""
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_equal, connect_nodes
+
+class EncryptionTest(BitcoinTestFramework):
+    def set_test_params(self):
+        self.setup_clean_chain = True
+        self.num_nodes = 4
+        self.extra_args = [['-netencryption'], ['-netencryption'], ['-netencryption=0'], ['-netencryption']]
+
+    def setup_network(self):
+        self.setup_nodes()
+        connect_nodes(self.nodes[0], 1)
+        connect_nodes(self.nodes[2], 1)
+        connect_nodes(self.nodes[2], 3)
+        connect_nodes(self.nodes[3], 1)
+        self.sync_all()
+
+    def getEncryptionSessions(self, node):
```

Comment:
Should be `get_encryption_sessions`? Could also be a function instead of method since `self` is not used?